site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+_**ElasticBeanstalkRollingUpdates**_  
+A collection of parameters that define how AWS Elastic Beanstalk performs rolling updates for the environment in which your application operates. For more information, see [Rolling updates](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.rollingupdates.html) in the [AWS Elastic Beanstalk Developer Guide](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/).
+
+This object is valid only if `EnvironmentType` (shown above) is set to "LoadBalanced".
+
+By default, rolling updates are disabled. To enable rolling updates, set `RollingUpdatesEnabled` to `true`. Then use the other parameters of this object (`RollingUpdateType`, etc.) to configure the behavior. These parameters are described as follows:
+
+* `RollingUpdateType`  
+The type of rolling update to implement.  
+-- Use a value of "Time" (the default) for time-based rolling updates. These updates wait a certain amount of time between batches. The amount of time is specified by the `PauseTime` parameter shown below.  
+-- Use a value of "Health" for health-based rolling updates. These updates wait for new instances to pass health checks before moving on to the next batch.  
+-- Use a value of "Immutable" for immutable updates. These updates launch a full set of instances in a new Auto Scaling group.
+
+* `MaxBatchSize`  
+The number of instances included in each batch of the rolling update. The valid range of values is 1 through 10000.
+
+* `MinInstancesInService`  
+The minimum number of instances that must be in service within the Auto Scaling group while other instances are terminated. The valid range of values is 0 through 9999.
+
+* `PauseTime`  
+The amount of time that the Elastic Beanstalk service waits after completing updates to one batch of instances before it continues with the next batch. The format of this parameter must be "PT#H#M#S" where each '#' is the number of hours, minutes, and seconds, each of which is optional.
+
+* `Timeout`  
+The maximum amount of time to wait for all instances in a batch of instances to pass health checks before canceling the update. The format of this parameter must be PT#H#M#S where each '#' is the number of hours, minutes, and seconds, each of which is optional. The default value is "PT30M".
+
+_**CNamePrefix**_  
+The prefix for the CNAME in your Elastic Beanstalk environment URL. If this parameter isn't specified or its value is an empty string (the default), the CNAME is generated automatically by appending a random alphanumeric string to the environment name.
+
+_**ElasticBeanstalkEnvironmentVariables**_  
+Environment properties for your application.
+
+_**VpcId**_  
+The ID of an existing Amazon Virtual Private Cloud (VPC) into which you want to launch your application. For more information about VPCs, see the [Amazon Virtual Private Cloud User Guide](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).

```

Comment:
What happens if they leave the string empty or leave the parameter out (DefaultValue of 'null')?
Will it throw an error or just use the Default VPC for the account?

Reply:
It wont throw an error. It just wont apply a custom VPC to the deployment. So if a VPC is needed, the default will be used.

Reply:
Thanks. Added info.